tests/bundle-test.js

```diff
@@ -219,6 +221,31 @@ async function runAll() {
       throw new Error('Claim content inside Req4Att mismatching')
   }
 
+  console.log('Test Messaging with encryption + decryption')
+  const body = {
+    content: {
+      requestForAttestation: signed,
+    },
+    type: MessageBodyType.REQUEST_ATTESTATION,
+  }
+
+  const message = new Message(body, bob.uri, alice.uri)
+  const encryptedMessage = await message.encrypt(
+    bob.encryptionKey.id,
+    bob,
+    keystore,
+    `${alice.uri}#${alice.encryptionKey.id}`
```

Comment:
we have
```suggestion
    alice.assembleKeyUri(alice.encryptionKey.id)
```
but still, this is crappy UI, you're right

Reply:
yeah, I think we need some good defaults for this